examples/grpc/grpc_foo_lib/foo_split.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include <sstream>
+#include <string>
+#include <vector>
+#include <map>
+#include "../tracer_common.h"
```

Comment:
This relative path `../` looks a bit unclean. You'd rather add the parent directory to include paths.

Reply:
removed this code, as we don't need this for simple example.